lib/perl/Genome/Model/Tools/ImportAnnotation/UpdateAnnotation.pm

```diff
@@ -64,8 +64,8 @@ sub execute
     my $flatfile = $self->derive_flatfile_location();
     my $gbff = "/gscmnt/sata835/info/medseq/annotation_data/human.rna.gbff"; # hard code this for now
 
-    my $ensembl_host = "mysql1"; # or mysql2?
-    my $ensembl_user = "mse";
+    my $ensembl_host = Genome::Config::get('db_ensembl_host');
+    my $ensembl_user = Genome::Config::get('db_ensembl_user');
```

Comment:
It does look old! This file's last update was in August 2009.  I only changed it because it was listed in #1546.

I agree that nobody should ever run this.  I'll take it out of this PR and remove it instead.